packages/genesys-spark-components/src/components/stable/gux-datepicker/gux-datepicker.service.ts

```diff
@@ -187,3 +188,37 @@ export function getIntervalRange(
 
   return { selectionStart, selectionEnd };
 }
+
+export function getFormat(component: HTMLElement): string {
+  const fixedDate = new Date('December 31, 2000 15:00:00 UTC+00:00');
```

Comment:
There may be an issue with this date and timezones. ie this moment in time is Jan 1st in australia